build_rhel.sh

```diff
@@ -6,7 +6,8 @@
 
 # The new gcc 7 is now available using the command 'source scl_source enable devtoolset-7'
 # the previous gcc will be enabled again after this script exits
-source scl_source enable devtoolset-7
+# Commented scl_source  because it does not work in Centos Stream 9
```

Comment:
scl_source is no longer needed with newer CentOS/RedHat systems, so these can be removed